ppr-ui/src/views/amendment/ConfirmAmendment.vue

```diff
@@ -426,97 +414,37 @@ export default defineComponent({
+      let docId = ''
       if (!localState.validFolio) {
-        const component = document.getElementById('folio-summary')
-        await component.scrollIntoView({ behavior: 'smooth' })
-        return
-      }
-      if (localState.registeringOpen) {
-        const component = document.getElementById('reg-party-change')
-        await component.scrollIntoView({ behavior: 'smooth' })
-        return
+        docId = 'folio-summary'
+      } else if (localState.registeringOpen) {
+        docId = 'reg-party-change'
+      } else if (!localState.courtOrderValid) {
+        docId = 'court-order-component'
+      } else if (!localState.certifyInformationValid) {
+        docId = 'certify-information'
       }
-      if (!localState.courtOrderValid) {
-        const component = document.getElementById('court-order-component')
+      if (docId) {
+        const component = document.getElementById(docId)
         await component.scrollIntoView({ behavior: 'smooth' })
-        return
-      }
-      if (!localState.certifyInformationValid) {
-        const component = document.getElementById('certify-information')
-        await component.scrollIntoView({ behavior: 'smooth' })
-      }
-    }
-
-    const loadRegistration = async (): Promise<void> => {
-      if (!localState.registrationNumber || !getConfirmDebtorName) {
-        if (!localState.registrationNumber) {
-          console.error('No registration number this amendment. Redirecting to dashboard...')
-        } else {
-          console.error('No debtor name confirmed for this amendment. Redirecting to dashboard...')
-        }
-        goToDash()
-        return
-      }
-
-      if (localState.currentRegNumber === localState.registrationNumber) {
-        return
       }
-
-      localState.financingStatementDate = new Date()
-      localState.submitting = true
-      const financingStatement = await getFinancingStatement(
-        true,
-        localState.registrationNumber
-      )
-
-      if (financingStatement.error) {
-        localState.dataLoadError = true
-        emitError(financingStatement.error)
-      } else {
-        await setStore(financingStatement)
-        // give time for setStore to finish
-        setTimeout(() => {
-          setUnsavedChanges(false)
-        }, 200)
-      }
-      localState.submitting = false
-    }
-
-    const setStore = async (financingStatement: FinancingStatementIF): Promise<void> => {
-      // load data into the store
-      const registrationType = AllRegistrationTypes.find((reg, index) => {
-        if (reg.registrationTypeAPI === financingStatement.type) {
-          return true
-        }
-      })
-      const parties = {
-        valid: true,
-        registeringParty: null, // will be taken from account info
-        securedParties: financingStatement.securedParties,
-        debtors: financingStatement.debtors
-      } as AddPartiesIF
-      setRegistrationCreationDate(financingStatement.createDateTime)
```

Comment:
The same state is set again when we go to review screen, which I don't believe is necessary since the state should already be in the store.